multi_gpu/README.md

```diff
@@ -0,0 +1,42 @@
+# Multi-GPU training example
+
+This example uses a CNN implemented in PyTorch to demonstrate how one might parallelize model training across several GPUs on a single node. Note that this is a different problem than  spreading the model across multiple GPUs, as might be required when training very large models that cannot fit in the memory of a single GPU. Moreover, this example demonstrates that multi-GPU training does not necessarily lead to speedups in training, and some thought should be given to whether a model will benefit from parallel training, as well as how that parallelism is implemented. 
```

Comment:
```suggestion
This example uses a CNN implemented in PyTorch to demonstrate how one might parallelize model training across several GPUs on a single node.
Note that this is a different problem than spreading the _model_ across multiple GPUs, as might be required when training very large models that cannot fit in the memory of a single GPU.
Moreover, this example demonstrates that multi-GPU training does not necessarily lead to speedups in training, and some thought should be given to whether a model will benefit from parallel training, as well as how that parallelism is implemented. 
```
Let's wait to see whether the parallel training gives speedups.  The way it is implemented may have a big impact.